src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type MovieType = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC <MovieType> = ({ movies }) => (
+  <ul className="movies">
+    {movies.map((movie) => (
+      <li className="card" key={movie.imdbId}>
```

Comment:
move ta li inside the MovieCard component instead of fragment

Reply:
Not resolved